mitosheet/mitosheet/api/get_imported_files_and_dataframes_from_current_steps.py

```diff
@@ -86,6 +87,15 @@ def get_import_data_with_single_import_list(step_type: str, params: Dict[str, An
             }
         }]
 
+    if step_type == UserDefinedImportStepPerformer.step_type():
+        return [{
+            'step_type': step_type,
+            'params': {
+                'importer': params['importer'],
+                'importer_params': params['importer_params']
+            }
+        }]
```

Comment:
I've had issues in some refactors where the fact that the custom imports can return multiple dataframes causes big time problems...

Let's just remove this feature. One custom import returns one function! Wanna update the customer importer code, and update tests? Should be pretty easy, and I don't think anyone is relying on this.